scripts/models/searchModel.js

```diff
@@ -0,0 +1,38 @@
+(function(module) {
+  var searchTool = new Object();
+  searchTool.AllCars = [];
+
+  searchTool.init = function(ctx, next) {
+    next();
+  };
+
+  searchTool.getAllCars = function(callback) {
+    $.get('/vehicle/v2/makes').done(function(data) {
+      callback(data);
```

Comment:
The functionality here is constructed in a bit of a 'weird' way. I would reconsider the workflow, and rather than having a callback to push the data into an array, just do that here in getAllCars.